api/src/pcapi/settings.py

```diff
@@ -500,3 +500,6 @@
 
 # ROUTES FOR END-TO-END TESTS
 ENABLE_TEST_ROUTES = bool(int(os.environ.get("ENABLE_TEST_ROUTES", 0)))
+
+# GOOGLE MAPS
+GOOGLE_MAPS_API_KEY = os.environ.get("GOOGLE_MAPS_API_KEY")
```

Comment:
Une clé d'API étant un secret, je pense que tu voulais plutôt : 
```suggestion
GOOGLE_MAPS_API_KEY = secrets_utils.get("GOOGLE_MAPS_API_KEY")
```